### PR TITLE
#161254625 like dislike article

### PR DESCRIPTION
**Pull request by Eyansky**
#### What does this PR do?
- enables a user to like or dislike an article. 

#### Description of Task to be completed?
- Lets appreciate those authors who go out of their way to create good content with a Like and encourage those authors who do not meet our expectations with a dislike.

#### How should this be manually tested?
- git clone this repo
```
$ git clone https://github.com/andela/ah-orcas-frontend
```
- Navigate to this branch
```
git checkout ft-like-dislike-#161254625
```
- Install the node modules
```
$ npm install
```
- Start the server
```
$ npm start
```
- It should open your browser to localhost port 3000.
- Log in via the login page then navigate to a single article.
- You should find two icons fro like and dislike. 
- Try to use them and see if they work as expected.

#### Any background context you want to provide?
- This was only to add the functionality for like and dislike.

#### What are the relevant pivotal tracker stories?
[#161254625](https://www.pivotaltracker.com/story/show/161254625)
#### Screenshots (if appropriate)
- How they look.
screenshot 2018-12-18 at 19 14 29
- As you wait for dislike to process.
screenshot 2018-12-18 at 19 15 34
- As you wait for like to process.
screenshot 2018-12-18 at 19 15 09
- How they look with an article
screenshot 2018-12-20 at 15 47 24

#### Checklist:

- [x] My code follows the style guidelines of this project
- [x] At least 2 people have reviewed my PR
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] My PR has one commit.

### CODE REVIEW

File: src/components/LikeDislike/__tests__/likedislike.test.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+              break;
+            
+            default:
+              throw Error('Unexpected render');
+          }
+        });
+    
+      });
+
+  });
```
[the remark this answers is not in the document]
Comment (Eyansky) [reply src/components/LikeDislike/__tests__/likedislike.test.js]: This has been fixed

File: src/actions/likes.actions.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  type: likeConstants.LIKES_SUCESS,
+  count: count.response,
+});
+
+export const dislikeSuccess = count => ({
+  type: likeConstants.DISLIKE_SUCESS,
+  count: count.response,
+});
+
+const token = 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJleHAiOjE1NDUzMTQ4MTMsImlhdCI6MTU0NDcxMDAxMywiaWQiOjUxfQ.qB3oqQNnoeoYnaTRIp9h4TXAmARQLfUjVVZzUpjAfCw';
```
Comment (TheSteelGuy) [inline src/actions/likes.actions.js]: Kindly pick the token from the logged in user and this pr will be good to go..

**Review by TheSteelGuy — CHANGES_REQUESTED** [review]

File: src/actions/likes.actions.js (hunk 2 above)
Comment (Eyansky) [reply src/actions/likes.actions.js]: This has been resolved.

**Review by TheSteelGuy — APPROVED** [review]
LGTM

**Review by Gidraf — APPROVED** [review]

**Pull request merged**